resources/js/app.js

```diff
@@ -28,6 +28,10 @@ import VueClipboard from 'vue-clipboard2';
 VueClipboard.config.autoSetContainer = true;
 Vue.use(VueClipboard);
 
+// Dependency of vuejs-clipper
+import VueRx from 'vue-rx';
+Vue.use(VueRx);
+
```

Comment:
I don't see where the plugin is used? If it's not used globally, is it good enough  to add it in the SetAvatar component?